core/sync/sync_util.odin

```diff
@@ -6,9 +6,10 @@ Example:
 		...
 	}
 */
-guard :: proc{
+guard :: proc {
```

Comment:
Why did you add random spaces?

Reply:
That was my VSCode auto-formatting, sorry.